app/stac_api/migrations/0047_prefill_counter_tables.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 5.0.7 on 2024-07-23 15:14
+
+from django.db import migrations
+
+
+# The migration SQL in this file was created manually.
+class Migration(migrations.Migration):
+    migrate_sql = '''
```

Comment:
From offline discussion: It's unlikely that these count tables will get out of sync, cause all triggers are atomic within a transaction. However, it's useful to have a mgmt command ready to reset/recalculate these values in case things get out of sync for some reason